src/usr/local/www/services_unbound_advanced.php

```diff
@@ -369,7 +369,7 @@
 	'disable_auto_added_access_control',
 	'Disable auto-added access control',
 	'disable the automatically-added access control entries',
-	$pconfig['hdisable_auto_added_access_control']
+	$pconfig['disable_auto_added_access_control']
```

Comment:
Thanks Phil,

Rant very much understood. I have learned that “converted” in the context of this project often meant “Looks pretty but completely untested”. You know the issues as well as (perhaps better) than me. My goal is to have the functionality restored to all pages by the end of this week, and then to start a month-long code review/QC process. I can’t tell you how much I value your help in this. It’s priceless.

> On Nov 10, 2015, at 1:18 AM, Phil Davis <notifications@github.com> wrote:
> 
> In src/usr/local/www/services_unbound_advanced.php <https://github.com/pfsense/pfsense/pull/2050#discussion_r44373387>:
> 
> > @@ -369,7 +369,7 @@
> >  	'disable_auto_added_access_control',
> >  	'Disable auto-added access control',
> >  	'disable the automatically-added access control entries',
> > -	$pconfig['hdisable_auto_added_access_control']
> > +	$pconfig['disable_auto_added_access_control']
> Need to have a rant here. What in the *** happened to code review/test in the original repo where this bootstrap stuff was done??? There are bugs like this all through the code - hundreds have been got out so far and who knows how many more there are to be found.
> 2.2.5 is a good piece of software, and 2.3 will be a PoC unless every line of code is re-checked to find all the dumb bugs introduced by the bootstrap rush.
> End of rant
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/pfsense/pfsense/pull/2050/files#r44373387>.
>